shared/src/test/diff/qq/Hygiene.mls

```diff
@@ -12,7 +12,10 @@ run(code_test)
 
 // Test 2: this is hygienic because the child Quoted is a descendant of the parent Quoted in the AST
 fun test(n, cde) = if n > 0 then code"let x = ${Const(n)} in ${test(n - 1, code"${cde} + x")}" else cde
-//│ fun test: forall 'x 'a. (Int, Code[Int, 'x] & 'a) -> (Code[Int, 'x] | 'a)
+//│ fun test: forall 'a 'b 'c. (Int, Code[Int, (??x | 'b) & 'a] & 'c) -> (Code[Int, ??x0 | 'a | 'b] | 'c)
```

Comment:
You should keep a run test for this function (as it originally had) to make sure the type is right.